image/image.go

```diff
@@ -29,11 +29,11 @@ func IDFromDigest(digest digest.Digest) ID {
 
 // V1Image stores the V1 image configuration.
 type V1Image struct {
-	// ID a unique 64 character identifier of the image
+	// ID is a unique 64 character identifier of the image
 	ID string `json:"id,omitempty"`
 	// Parent id of the image
```

Comment:
Looking at this again, I think I know why these comments were inconsistent. I think the original intent of these was to write;

```
ID - a unique 64 character identifier of the image
Comment - user added comment
```

Which doesn't follow GoDoc conventions. _However_, now that _some_ follow those conventions, and some don't, we should be careful;

```
Parent id of the image
```

could now be interpreted as

```
Parent - ID of the image
```

Which isn't correct. I suggest (but further suggestions welcome);

```
Parent is the ID of the parent image
Comment is the commit message that was set when committing the image
Created is the timestamp at which the image was created
DockerVersion specifies the version of Docker that was used to build the image
Author is the name of the author that was specified when committing the image
```